R/convert_ov_wenr.R

```diff
@@ -0,0 +1,37 @@
+#' Convert possible B_OV_WENR values to standardised values
+#' 
+#' This function converts numeric values for OV_WENR to values used by other OBIC functions if numeric values are entered.
+#' 
+#' @param B_OV_WENR () Data on soil compaction risk that may have to be converted to string
```

Comment:
Add in the brackets what type of input you expect (numeric and/or character)

Reply:
Are there any reserved words/phrases for in between brackets? or is it just informative with (numeric or character) being acceptable?

Reply:
No, these are not reserved. To be consistent with other function use `character` and `numeric`